gips/data/prism/prism.py

```diff
@@ -235,26 +233,33 @@ def process(self, *args, **kwargs):
             missingassets = []
             availassets = []
 
+
             vsinames = {}
 
-            def get_bil_file(d, a):
-                return filter(
-                    lambda f: f.endswith('.bil'),
-                    d.assets[a].datafiles(),
-                )[0]
+            def get_bil_vsifile(d, a):
+                b = None
```

Comment:
optional:  functions in python `return None` implicitly, so I think you can just `return os.path.join(...)` and simplify things a bit.